annotation/tests/test_jobs.py

```diff
@@ -0,0 +1,130 @@
+import uuid
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from annotation.errors import (
+    EnumValidationError,
+    FieldConstraintError,
+    WrongJobError,
+)
+from annotation.jobs.services import check_annotators, update_inner_job_status
+from annotation.models import Category, File, Job, User
+from annotation.schemas import (
+    CategoryTypeSchema,
+    FileStatusEnumSchema,
+    JobStatusEnumSchema,
+    TaskStatusEnumSchema,
+    ValidationSchema,
+)
+
+JOB_TENANT = "test"
+ANNOTATORS = (
+    User(user_id="82533770-a99e-4873-8b23-6bbda86b59ae"),
+    User(user_id="ef81a4d0-cc01-447b-9025-a70ed441672d"),
+)
+CATEGORIES = [
+    Category(
+        id="18d3d189e73a4680bfa77ba3fe6ebee5",
+        name="Test",
+        type=CategoryTypeSchema.box,
+    )
+]
+
+JOB_IDS = [1, 2, 3]
+JOBS_TO_TEST_PROGRESS = (
+    Job(  # Annotation job with 3 tasks in progress, 1 finished task
+        job_id=JOB_IDS[0],
+        callback_url="http://www.test.com",
+        annotators=[
+            ANNOTATORS[0],
+            ANNOTATORS[1],
+        ],
+        validation_type=ValidationSchema.cross,
+        is_auto_distribution=False,
+        categories=CATEGORIES,
+        deadline="2021-10-19T01:01:01",
+        tenant=JOB_TENANT,
+    ),
+    Job(  # Extraction job with no tasks
+        job_id=JOB_IDS[1],
+        callback_url="http://www.test.com",
```

Comment:
Could you make this testing data as a fixture?

Reply:
I made the testing data as a fixture, but like this i can't use pytest.parametrize for "annotators" parameter in test_check_annotators.

Reply:
That's why you probably dont need a fixture and constant for this. create a simple user is not a big deal, `User(user_id="82533770-a99e-4873-8b23-6bbda86b59ae")`